RIPS/rip-7560.md

```diff
@@ -276,14 +275,12 @@ The `txHash` parameter represents the hash of the AA_TX_TYPE transaction with em
 The frame must return a bytes array that is interpreted as:
 
 ```solidity
-
-abi.encode(context, MAGIC_VALUE_PAYMASTER, validUntil, validAfter)
-
+MAGIC_VALUE_PAYMASTER <20 bytes>, validUntil <6 bytes>, validAfter <6 bytes>, 64 <constant, 32 bytes>, length <32 bytes>, context <variable>
 ```
+(this is the same encoding the the above solidity method returns)
 
 Same as in the [`sender` validation frame](#sender-validation-frame), in order to support gas estimation this
-frame should return a value different from `MAGIC_VALUE_PAYMASTER` for conditions that cannot be satisfied
-before signing.
+frame should revert on errors, but return `MAGIC_VALUE_SIGFAIL` if the failure is because of signing error.
```

Comment:
"in order to support gas estimation this frame should return `MAGIC_VALUE_SIGFAIL` if the failure is only caused by a signature verification error. This frame can revert in case of other exceptions." Something like that?

Reply:
I still think this phrase could be rephrased to be clearer.

Reply:
then please rephrase.